test/config.test

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+var config = require('./config.global');
```

Comment:
File is named `config.test` instead of `config.js` because it will fail because that required file is not in `test` directory.